app/scripts/proactive/view/CatalogGetView.js

```diff
@@ -56,8 +57,9 @@ define(
                 var bucketName = that.getSelectedBucketName();
                 var filterKind = this.kind;
                 //for workflows, we don't want subkind filters (ie we want to be able to import workflow/pca and workflow/standard)
-                if (this.kind.toLowerCase().indexOf('workflow') > -1)
-                    filterKind = "workflow"
+                if (this.kind.toLowerCase().indexOf('workflow') > -1) {
```

Comment:
not sure this is robust as you search for workflow anywhere in the string.
If the workflow kind are:
workflow/standard and workflow/pca

To avoid conflicts with user-defined kinds, it would probably be better to use:
if (this.kind.toLowerCase().indexOf('workflow') == 0) {
}
We know the workflow kinds starts with workflow

Reply:
You're right, I'll change that